pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [project]
 name = "detection_rules"
-version = "1.0.15"
+version = "1.1.15"
```

Comment:
```suggestion
version = "1.1.0"
```

Reply:
Unless there is a specific reason to bump to .15 which I don't see. I think, we can check on this when you are online and I will add an additional review so we can close this.